cloudigrade/config/settings/base.py

```diff
@@ -634,6 +634,15 @@ def __print(*args, **kwargs):
     "ENABLE_SYNTHETIC_DATA_REQUEST_HTTP_API", default=False
 )
 
+#####################################################################
+# leader run cached timing related configs for features
+
+# How long for a feature to expect to be running as a leader for
+LEADER_RUNNING_TTL_DEFAULT = env.int("LEADER_RUNNING_TTL_DEFAULT", default=60)
+
+# How long do we take a feature completed as a leader for
```

Comment:
What do you mean my "feature" and "feature completed" in these comments?

Reply:
And also in the actual code below in `leaderrun`.

I think I understand what it's doing. I'm just a little puzzled by the language around it. Was `Feature` something you copied from elsewhere? Do the various docstrings and variable names need to be updated to match correctly if so?

Reply:
Naming is hard, not copied from anywhere, is the "thing", the "piece of code", the "capability", the "widget" of cloudigrade that we're running as a leader since the helper class allows us to run many different things as a leader. I guess maybe just "leaderkey" ? I'm open to suggestions. 🤷